Library/ViewModels/ErroredBackingViewViewModel.swift

```diff
@@ -37,9 +45,65 @@ public final class ErroredBackingViewViewModel: ErroredBackingViewViewModelType,
     self.manageButtonTappedObserver.send(value: ())
   }
 
+  public let finalCollectionDateText: Signal<String, Never>
   public let notifyDelegateManageButtonTapped: Signal<GraphBacking, Never>
   public let projectName: Signal<String, Never>
 
   public var inputs: ErroredBackingViewViewModelInputs { return self }
   public var outputs: ErroredBackingViewViewModelOutputs { return self }
 }
+
+private func timeLeftString(date: String) -> String {
```

Comment:
I think the return type here should be optional, a `nil` value better represents it not having returned anything than an empty string.